roadmap/implementers-guide/src/runtime/paras.md

```diff
@@ -1,12 +1,16 @@
 # Paras Module
 
-The Paras module is responsible for storing information on parachains and parathreads. Registered parachains and parathreads cannot change except at session boundaries. This is primarily to ensure that the number of bits required for the availability bitfields does not change except at session boundaries.
+The Paras module is responsible for storing information on parachains and parathreads. Registered
+parachains and parathreads cannot change except at session boundaries. This is primarily to ensure
+that the number of bits required for the availability bitfields does not change except at session
```

Comment:
why would the number of bitys alter when upgrading/downgrading between parthread and parachain?

Reply:
Every availability core gets its own bit in the bitfield. Parathreads are multiplexed over a fixed number of cores (configured per-session), whereas Parachains each get their own dedicated core. Upgrading from parathread to parachain would alter the amount of availability cores, which would alter the division of validators into groups, which could lead to consensus issues across forks.